code/modules/mob/living/basic/basic.dm

```diff
@@ -6,15 +6,22 @@
 	maxHealth = 20
 	gender = PLURAL
 	living_flags = MOVES_ON_ITS_OWN
-	status_flags = CANPUSH
+	status_flags = CANPUSH | CANSTUN
 	fire_stack_decay_rate = -5 // Reasonably fast as NPCs will not usually actively extinguish themselves
 
 	var/basic_mob_flags = NONE
 
+	/// What's the maximum amount of stamina we can lose? By default, same as maximum health
+	var/max_stamina_loss = BASIC_MOB_STAMINA_MATCH_HEALTH
```

Comment:
``/mob/living`` already implements ``max_stamina``, I'm pretty sure this is redundant